### DISCUSSION TITLE
LQAS: Child Fingermarked with Reason of Missed

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#670] [General]
As A rule If Fingermarked so it should not display/ record the missed reason and vise versa.

@bnygren @RaziaMahmodi 

![image](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/419b5fec-65da-4b3d-b85b-3b4f033bb559)

![image](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/85d96a42-2a2d-423b-929b-c25f6cb10167)

**Comment by bnygren** [comment]
To confirm, in the form, users are having the reason displayed even when fingermarked = 'NO'? i.e., should be conditional on fingermarked = 'YES' 
Thanks.
